pkg/minikube/cluster/cluster.go

```diff
@@ -601,3 +605,33 @@ func getKubernetesServicesWithNamespace(namespace string) (serviceGetter, error)
 	services := client.Services(namespace)
 	return services, nil
 }
+
+func GetKubernetesEndpointsWithNamespace(namespace string) (endpointGetter, error) {
+	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
```

Comment:
Could you refactor out the creation of the kubeConfig.ClientConfig? This and GetKubernetesServicesWithnamespace could both use it.